gapic/generator/generator.py

```diff
@@ -297,6 +297,8 @@ def _render_template(
                         # TODO(yon-mg) - remove when rest async implementation resolved
                         # temporarily stop async client gen while rest async is unkown
                         ('async' in template_name and 'grpc' not in opts.transport)
+                        or
```

Comment:
Which line are you referring to?

`('async' in template_name and 'grpc' not in opts.transport)` is from before and it's guarding async client generation i.e. `async_client.py` for rest (since we don't support rest transport for async clients yet). We should remove it once we do (as the comment states) and instead guard the client generation based on the optional feature client option that we're adding.

`('rest_base' in template_name and 'rest' not in opts.transport)` is what I added to guard the generation of `rest_base.py` if rest is not listed as a required transport. I don't see this being removed unless we find a better way to do it.